charts/s1-agent/templates/_helpers.tpl

```diff
@@ -170,3 +169,19 @@ Create the name of the service account to use
 {{- define "service.target_port" -}}
 {{- print 6443 -}}
 {{- end -}}
+
+{{- define "helper.full_url" -}}
+{{- if .Values.configuration.image.helper -}}
+{{ .Values.configuration.image.helper }}
+{{- else -}}
+{{ required "Must set the appropriate registry for agent helper pulling" .Values.configuration.repositories.helper }}:{{ default .Values.configuration.tag.agent .Values.configuration.tag.helper }}
```

Comment:
shouldn't it be required as same as the agent?

Reply:
The logic here (same as always was) is to take the agent's tag if the helper tag wasn't provided. I think it is fine to leave it this way, since users might depend on that already